src/coreComponents/physicsSolvers/fluidFlow/wells/WellControls.hpp

```diff
@@ -226,18 +220,30 @@ class WellControls : public dataRepository::Group
    */
   const real64 & getSurfaceTemperature() const { return m_surfaceTemp; }
 
+  /**
+   * @brief Getter for the well type
+   * @return true if the well is an injector, false otherwise
+   */
+  bool isInjector() const { return ( m_type == Type::INJECTOR ); }
+
+  /**
+   * @brief Getter for the well type
+   * @return true if the well is a producer, false otherwise
+   */
+  bool isProducer() const { return ( m_type == Type::PRODUCER ); }
+
   /**
    * @brief Getter for the status of the well (open or shut)
    * @param[in] currentTime the current time
    * @return a flag equal to true if the well is open, and false otherwise
    */
-  bool wellIsOpen( real64 const & currentTime ) const;
+  bool isWellOpen( real64 const & currentTime ) const;
 
   /**
    * @brief Getter for the flag to disable crossflow in injectors
    * @return the flag deciding whether crossflow is allowed or not
    */
-  bool disableInjectorCrossflow() const { return m_disableInjectorCrossflow; }
+  bool isCrossflowDisabled() const { return m_isCrossflowDisabled; }
```

Comment:
Just a detail, but it could be a good habit to use some
```suggestion
  bool isCrossflowEnabled() const { return m_isCrossflowEnabled; }
```
to prevent from some double negations like `wellControls.isInjector() && !wellControls.isCrossflowDisabled()`.
And would a `wellControls.hasCrossFlow()` would make sense? (not sure...)

Reply:
Done, I changed to `isCrossflowEnabled()`.

For the `wellControls.hasCrossFlow()` getter, it makes less sense, because it is something that we detect on the fly, so there is no need for a getter.